src/AbstractSQLRules2SQL.ts

```diff
@@ -889,6 +910,19 @@ const typeRules: Dictionary<MatchFn> = {
 		const replacement = TextValue(getAbstractSqlQuery(args, 2), indent);
 		return `REPLACE(${str}, ${find}, ${replacement})`;
 	},
+	ExtractJSONPathAsText: (args, indent) => {
+		checkMinArgs('ExtractJSONPathAsText', args, 1);
+		if (engine !== Engines.postgres) {
+			throw new SyntaxError(
+				'ExtractJSONPathAsText not supported on: ' + engine,
+			);
+		}
+		const column = TextValue(getAbstractSqlQuery(args, 0), indent);
+		const path = args[1]
+			? TextValue(getAbstractSqlQuery(args, 1), indent)
+			: `CAST(ARRAY[] as TEXT[])`;
```

Comment:
The typings say that the path must always be specified:
```suggestion
		const path = TextValue(getAbstractSqlQuery(args, 1), indent);
```